Wireframe/index.html

```diff
@@ -3,11 +3,49 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <meta name="description" content="What is Git.">
+    <meta name="keywords" content="Git, Branch, Developers, Version Control">
+    <meta name="author" content="Pooriya">
+    <title>Git</title>
+    <link rel="stylesheet" href="./css/style.css">
   </head>
   <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+      <header>
+        <h1>Get to know <img id="logo" src="../IMG/Git-Logo-1788C.svg" alt="Git Logo"></h1>
+        <h2>Version Control for Modern Development</h2>
+      </header>
+      <main class="article">
+        <article id="first">
+            <h3>What is Git?</h3>
+            <img id="git-img" src="../IMG/git.png" alt="">
```

Comment:
What is the purpose of the `alt` prop here? What does an empty string do?

Reply:
You're right! I forgot to add a description to the alt attribute. I'll update it with descriptive text to improve accessibility. 
Thanks for catching that!